lua/entities/gmod_wire_expression2/core/extloader.lua

```diff
@@ -14,14 +14,14 @@ if ENT then
 			ply:PrintMessage( 2, "Sorry " .. ply:Name() .. ", you don't have access to this command." )
 			return
 		end
-		
+
```

Comment:
@AbigailBuccaneer @Divran do you mind if i kill all the trailing whitespace once and for all? :)